### DISCUSSION TITLE
Is there a limit for how many ticks we can have in the x-axis?

### DISCUSSION THREAD

**Discussion by Tejada-Joyce** [plouc/nivo#2404] [Q&A] [answered]
I'm using a Bar Chart with a date time range (which technically works as regular data since I'm passing a value for each day), and as the range increases, there are times when the ticks start to "condense" in the center instead of occupying the whole "line".
Example:
From: 01/05 - 07/28
<img width="1317" alt="image" src="https://github.com/plouc/nivo/assets/35240350/204ce157-599b-4d1d-aa0d-739e1bd7a96a">
From: 01/01 - 07/28
<img width="1317" alt="image" src="https://github.com/plouc/nivo/assets/35240350/94206b84-a471-4b5a-8111-35479ea8b58d">

**Comment by SebasQuirogaUCP** [comment]
There is a prop on AxisProps `tickValues` that can help you with it [Nivo Documentation](https://nivo.rocks/guides/axes/)

![image](https://github.com/plouc/nivo/assets/28678404/c6ee693d-e56c-4796-b93c-6f26d48e0eba)

```
axisBottom={{
            tickValues: 5,
           ...            
            }}
```

**Comment by tkonopka** [comment, marked as answer]
The "condense in the center" effect in the screenshots is possibly related to #929. In particular, [this comment](https://github.com/plouc/nivo/issues/929#issuecomment-1192271248) summarizes why the effect happens and gives an adjustment to chart props (through `indexScale` and `round`).

Setting `tickValues` also affects space around ticks, although it is more often used with continuous axes.

**Comment by SebasQuirogaUCP** [comment]
@tkonopka Thank you for bringing that up  👍🏼
